umh-core/pkg/fsmv2/benthos_monitor/manager.go

```
@@ -0,0 +1,285 @@
+	}
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, 0, err
+	}
+	// Defensive: http.Client.Do may return a nil resp alongside a non-nil error
+	// on some paths; the error check above normally covers it, but guard the
+	// dereferences that follow so a nil resp can never slip through.
+	if resp == nil {
+		return nil, 0, fmt.Errorf("GET %s: nil response", url)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)